src/utils/utils.cpp

```diff
@@ -54,6 +54,12 @@ namespace
 {
 constexpr auto category = "utils";
 
+const QString client_cert_prefix{"multipass_cert"};
+const QString cert_file_suffix{".pem"};
+const QString key_file_suffix{"_key.pem"};
+const QString client_cert_file{client_cert_prefix + cert_file_suffix};
+const QString client_key_file{client_cert_prefix + key_file_suffix};
```

Comment:
Dupe from `ssl_cert_provider.h`, could make common?

Reply:
Oh, oops, good catch!  I think the one in `ssl_cert_provider.h` is cruft.  I'll remove that.

Reply:
And move these to a common place as you suggest :)

Reply:
Fixed.  It's all in `utils.h` now.